inc/issue.class.php

```diff
@@ -508,6 +508,7 @@ static function getDefaultSearchRequest() {
    }
 
    public static function giveItem($itemtype, $option_id, $data, $num) {
+      $ticket = new Ticket();
```

Comment:
You should move this line just before L525 to instanciate a Ticket only when it will be useful.